docs/docs/teams/add-team.md

```diff
@@ -1,11 +1,12 @@
 ---
+sidebar_position: 1
 title: Add a Team
 intro: Add teams to organize groups of people in any way that makes it more convenient to grant and revoke project permissions.
 links:
     overview: Project owners can create teams as a initial step to manage permissions and collaboration on environment resources.
     quickstart:
-    previous: ssl-certificates/add-ssl-certificate
-    next: teams/invitations/add-invitation
+    previous:
+    next:
```

Comment:
After add9ing a team the next logical step is to invite somebody for the team (add team members).

Maybe keep the value of `next` as it was previously defined?

Reply:
Fixed this one by adding `sidebar_position` property in the rest of the docs present under the `teams` section/topic. 

<img width="1209" height="515" alt="Screenshot 2025-08-27 at 3 26 31 AM" src="https://github.com/user-attachments/assets/9628c9db-275e-4559-b0d2-555bf2ed3898" />